### ISSUE TITLE
Long-running write

### ISSUE THREAD

**Issue by davidohana** [doyaaaaaken/kotlin-csv#59]
Please allow writing to csv file without having to close it.
I have a streaming scenario where I need to write each data I get to csv. Closing and reopening after each batch would be suboptimal.

Thanks
David

**Comment by doyaaaaaken** [comment]
@davidohana 
Thank you for your valuable input.

In that scenario, I guess you can write csv from streaming data like below code.
If I'm not understanding your situation correctly, please give me more details.

```
import com.github.doyaaaaaken.kotlincsv.dsl.csvWriter

//streaming data
val rows = sequenceOf(
    listOf("a", "b", "c"),
    listOf("d", "e", "f")
)

csvWriter().open("test.csv") {
    rows.forEach { row ->
        writeRow(row)
    }
}
```

**Comment by davidohana** [comment]
Hello, in the code above, the file handle is closed at the end of the block. I get new data the I need to write to CSV every few seconds and I want to keep writing it to an open file without reopening it.

**Comment by doyaaaaaken** [comment]
@davidohana 
Thanks.
So, does the method `openAndGetRawWriter` described below solve your problem?

```kotlin
val writer: CsvFileWriter = csvWriter().openAndGetRawWriter("test.csv")
writer.writeRow(listOf("a", "b", "c"))
writer.writeRow(listOf("d", "e", "f"))
writer.close()
```

**Comment by davidohana** [comment]
Yes it should, but I didn't see such a public method.

**Comment by doyaaaaaken** [comment]
@davidohana 
OK, I'll do it soon. If you'd like to implement it by yourself, feel free to tell me and send pull request.

**Comment by doyaaaaaken** [comment]
@davidohana 
released at v0.11.0, thanks.

**Comment by davidohana** [comment]
Thanks!
